core/src/main/scala/org/apache/spark/sql/rapids/tool/EventProcessorBase.scala

```diff
@@ -89,6 +90,10 @@ abstract class EventProcessorBase[T <: AppBase](app: T) extends SparkListener wi
       case _: SparkListenerSQLAdaptiveSQLMetricUpdates =>
         doSparkListenerSQLAdaptiveSQLMetricUpdates(app,
           event.asInstanceOf[SparkListenerSQLAdaptiveSQLMetricUpdates])
+      case _: StreamingQueryListener.QueryStartedEvent
+        | _: StreamingQueryListener.QueryTerminatedEvent =>
+        throw StreamingEventLogException(
+          "Encountered Spark Structured Streaming Job: skipping this file!")
```

Comment:
nit: we could chandle the exception in a new method `doSparkListenerStreamingQuery` in case we want to expand the implementation in the future.